ush/forecast_postdet.sh

```diff
@@ -663,14 +667,17 @@ WW3_nml() {
 }
 
 WW3_out() {
+  mkdir -p "${COM_CONF}"
   echo "SUB ${FUNCNAME[0]}: Copying output data for WW3"
 }
 
 
 CPL_out() {
+  mkdir -p "${COM_CONF}"
   echo "SUB ${FUNCNAME[0]}: Copying output data for general cpl fields"
   if [[ "${esmf_profile:-}" = ".true." ]]; then
-    ${NCP} "${DATA}/ESMF_Profile.summary" "${COM_ATMOS_HISTORY}/ESMF_Profile.summary"
+    ${NCP} "${DATA}/ESMF_Profile.summary" "${COM_ATMOS_HISTORY}/ESMF_Profile.summary" # TODO: I don't think this is needed for cycling.
```

Comment:
This is a runtime output, right? It is not an input or a configuration. 
Why is this useful to be saved?

Reply:
This is an output that's only created when esmf_profile = true, which should only be done for getting information about runtime performance of the run, and  should be false when in production.  This isn't needed for cycling but can be used in a cycling scenario.

Reply:
Done.

Reply:
@JessicaMeixner-NOAA Thank you for the clarification.